homeassistant/components/fronius/sensor.py

```diff
@@ -137,13 +147,27 @@ def device_state_attributes(self):
                           "Maybe the configured device is not supported")
 
         if values:
+            # Copy data of current fronius device
             self._state = values['status']['Code']
             attributes = {}
             for key in values:
                 if 'value' in values[key]:
-                    attributes[key] = values[key].get('value', 0)
+                    attributes[key] = values[key]
```

Comment:
If we add sensors for the keys we shouldn't add them as state attributes on this entity.